Automattic/newspack-popups
# Add/overlay more placements and size option

### All Submissions:






### Changes proposed in this Pull Request:



Closes #114.

### How to test the changes in this Pull Request:

1. Checkout the PR branch.
2. From the campaign menu, add a new popup prompt.
3. Observe the new placements selector and size options.
4. select any placement and size for the new popup and save it.
5. From the campaigns list, preview the newly created popup.
6. Observe the popup showing in the selected placement with the selected size.

### Some notes:

- The size option works for large screens only, for the small screens, it will always be at 100%.
- The popup size is set by default to full-width when the popup is centered on top or on the bottom.
- The sizes are: Small: 25%, Medium: 50%, Large: 75%, Full-width: 100%.

### Other information:
